pubspec.yaml

```diff
@@ -6,18 +6,18 @@ repository: https://github.com/dart-code-checker/dart-code-metrics
 issue_tracker: https://github.com/dart-code-checker/dart-code-metrics/issues
 
 environment:
-  sdk: '>=2.12.0 <3.0.0'
+  sdk: ">=2.12.0 <3.0.0"
 
 dependencies:
-  analyzer: ^1.4.0
+  analyzer: ^1.5.0
```

Comment:
Have you checked that this change will not lead to inability to resolve dependencies in dependant projects?